generator/lib/src/error.dart

```diff
@@ -43,7 +43,7 @@ class RealmInvalidGenerationSourceError extends InvalidGenerationSourceError {
         color = color ?? session.color,
         super(message, todo: todo, element: element) {
     if (element is FieldElement || element is ConstructorElement) {
-      final classElement = element.enclosingElement!;
+      final classElement = element.enclosingElement3!;
```

Comment:
Maybe should be element2.

Reply:
No - `enclosingElement3` is correct. `enclosingElement2` ids deprecated